src/app/components/password-reset/password-reset.component.ts

```diff
@@ -69,7 +69,7 @@ export class PasswordResetComponent implements OnInit {
               type: 'success',
               message: 'Password Reset success! Logging in...'
             };
-            this.router.navigateByUrl('/channels');
+            this.router.navigateByUrl('/campaigns');
```

Comment:
This is unfortunately not correct. Default page should still be channels. David only wanted the Campaigns to be renamed to "Crowdfunding" in the header.